test/metabase/test/generate.clj

```diff
@@ -150,7 +149,7 @@
 (s/def ::http-action (s/keys :req-un [::template]))
 
 (s/def ::core-user (s/keys :req-un [::id ::first_name ::last_name ::email ::password]))
-(s/def ::collection (s/keys :req-un [::id ::name ::color]))
+(s/def ::collection (s/keys :req-un [::id ::name]))
```

Comment:
This should fix failures in `e2e-storage-ingestion-test` for mysql and mariadb.
https://github.com/metabase/metabase/actions/runs/6356184320/job/17265431237?pr=34239